narwhals/translate.py

```diff
@@ -132,7 +132,6 @@ def from_native(
     *,
     pass_through: Literal[True],
     eager_only: Literal[False] = ...,
-    eager_or_interchange_only: Literal[True],
```

Comment:
this param _could_ be slowly deprecated...but tbh I'm pretty keen to simplify `from_native` (the first thing most people will see), and we're keeping this around in `stable.v1` anyway